src/Keboola/S3Extractor/S3AsyncDownloader.php

```diff
@@ -22,7 +22,7 @@ class S3AsyncDownloader
 {
     private const MAX_ATTEMPTS = 5;
     private const INTERVAL_MS = 500;
-    private const MAX_CONCURRENT_DOWNLOADS = 50;
+    private const MAX_CONCURRENT_DOWNLOADS = 200;
```

Comment:
Zbytocne stahujeme subory po `50`, maximum je `5500`, pri 200 to potrebuje `200MB` memory, ... velkost suboru by nemala mat zasadny vplyv, je tam iba nejaky buffer.